google/cloud/spanner/internal/session_pool.cc

```diff
@@ -297,11 +297,25 @@ Status SessionPool::CreateSessions(
 }
 
 StatusOr<SessionHolder> SessionPool::Allocate(bool dissociate_from_pool) {
+  return Allocate(std::unique_lock<std::mutex>(mu_), dissociate_from_pool);
+}
+
+std::shared_ptr<SpannerStub> SessionPool::GetStub(Session const& session) {
+  auto const& channel = session.channel();
+  if (channel) return channel->stub;
+
+  // Sessions that were created for partitioned Reads/Queries do
+  // not have their own channel/stub, so return a stub to use by
+  // round-robining between the channels.
```

Comment:
nit: I am not sure we will find "robining" in the dictionary, but that's ok.

Reply:
Ack.